music21/musicxml/m21ToXml.py

```diff
@@ -4403,12 +4450,31 @@ def noteheadToXml(self, n: note.NotRest) -> Element:
             mxNotehead.set('color', color)
         return mxNotehead
 
+    @staticmethod
+    def topAndBottomIndices(chordOrNote) -> t.Tuple[int, int]:
+        # Use sortAscending if you can, assume ascending if you can't.
+        # If chordOrNote is a note (not expected, but possible), (0, 0) will be returned.
+        # It would be nice if PercussionChord could sortAscending (ascending on the
+        # staff, that is)...
+        top: int = len(chordOrNote.notes)
+        bottom: int = 0
+
+        if isinstance(chordOrNote, chord.Chord):
+            ascendingChord = chordOrNote.sortAscending(inPlace=False)
```

Comment:
This can be done inPlace and then no need for the extra indexes, etc.

By the time we're exporting to musicxml there's already been a deepcopy done once, so everything is fair game to manipulate.

Reply:
done.